src/ray/common/cgroup2/BUILD.bazel

```diff
@@ -79,7 +79,33 @@ ray_cc_library(
     }),
 )
 
-# Private Targets.
+# Private targets
+#
+# TODO(#54703): This target builds the noop implementations.
+# There's a corressponding test that runs on Linux and Non-Linux
+# CI so breakages are caught in premerge before these targets are
+# cleaned up at the end of the resource isolation milestone 1
+# project.
```

Comment:
I feel we don't really need this given we are very close to remove the noop thing and I'm sure you won't miss it anymore.